xpk/README.md

```diff
@@ -115,3 +162,29 @@ feedback.
 * Workload create accepts a --env-file flag to allow specifying the container's
 environment from a file. Usage is the same as Docker's
 [--env-file flag](https://docs.docker.com/engine/reference/commandline/run/#env)
+
+* Access stable fleet TPU machines by adding `--host-maintenance-interval=PERIODIC`
```

Comment:
I don't think StableFleet is GA so I think we should remove this.

Reply:
> I don't think StableFleet is GA so I think we should remove this.

Done.